app.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const mongoose = require("mongoose");
+const app = express();
+
+const {Schema} = mongoose;
+
+const taskScheme = new Schema ({
+  text: String,
+  isChek: Boolean,
+});
+
+const Task = mongoose.model("tasks", taskScheme);
+
+app.use(cors());
+
+const url = "mongodb+srv://Vitaliy:ZAQwed123@cluster0.c5jod.mongodb.net/TodoList?retryWrites=true&w=majority";
+mongoose.connect(url, {useNewUrlParser: true, useUnifiedTopology: true});
```

Comment:
why you need it ? do you know what is it ? useNewUrlParser: true, useUnifiedTopology: true